src/components/Pill/PillButton/PillButton.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp, IoIosArrowDown } from 'react-icons/io';
+import { CROSS_CHAR } from '../../../constants';
+import styles from './PillButton.scss';
+
+const { block, elem } = bem({
+    name: 'PillButton',
+    classnames: styles,
+    propsToMods: ['isActive', 'isOpen'],
+});
+
+const PillButton = React.forwardRef((props, ref) => {
+    const { isOpen, togglePopup, onClear, label, contentLabel, ...rest } = props;
+    const isActive = !!contentLabel;
+    const propsForBem = { ...props, isActive };
+
+    const getButton = () => {
+        if (isOpen) return { icon: <IoIosArrowUp />, onClick: () => null };
+        if (isActive)
```

Comment:
Can we add pairs of curly brackets to these `if`s for better readability? It's always nice to have them I think, especially in `isActive` case